src/web/components/SharingPermission/searchAndAddParticipants.tsx

```diff
@@ -1,35 +1,31 @@
-import { Suspense, useContext, useMemo, useState } from 'react';
```

Comment:
Could we possibly rename these three files to be PascalCase, for consistency?

Reply:
done